services/order.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    user = User.objects.get(username=username)
```

Comment:
The `get` method can raise a `DoesNotExist` exception if no user is found with the given username. Consider handling this exception to prevent potential crashes.